### PR TITLE
fix(agw): Enabled flaky retry in CI

### PR DESCRIPTION
**Pull request by VinashakAnkitAman**
## Title
fix(agw): Enabled flaky retry for failing test cases in CI

## Summary
This PR enables flaky test case retry in CI runs

## Test plan
Verified with sanity

Signed-off-by: VinashakAnkitAman <ankit.aman@radisys.com>

### CODE REVIEW

**Review by sebathomas — APPROVED** [review]
It's good that flaky retries are no longer the default if you run them on your laptop. :+1:

**Review by ardzoht — COMMENTED** [review]
Can you also update `https://github.com/magma/magma/tree/master/lte/gateway/python/integ_tests` with two things:
1. `-i` flag to enable continuous test runs
2. `enable-flaky-retry` flag to perform retries

File: lte/gateway/python/integ_tests/Makefile [hunk 1]
```diff
@@ -24,9 +24,14 @@ $(PYTHON_BUILD)/setupinteg_env:
 
 # TODO T21489739 - Don't sleep and don't stop after a failure
 RESULTS_DIR := /var/tmp/test_results
+ifdef ENABLE-FLAKY-RETRY
```
Comment (ardzoht) [inline lte/gateway/python/integ_tests/Makefile]: can we define this as lowercase?

**Review by ardzoht — APPROVED** [review]
lgtm

**Review by themarwhal — APPROVED** [review]
doc changes lgtm!

**Comment by VinashakAnkitAman** [comment]
Hi @themarwhal .. Markdown lint check with insync was always failing on this PR. To fix that I had to commit changes in the file: docs/docusaurus/i18n/en.json. But this file does not include any of my change. Can you please cross-check if I need to commit this file?

**Comment by themarwhal** [comment]
> Hi @themarwhal .. Markdown lint check with insync was always failing on this PR. To fix that I had to commit changes in the file: docs/docusaurus/i18n/en.json. But this file does not include any of my change. Can you please cross-check if I need to commit this file?

Yes those are good to commit. I will look into why this did not fail on the actual PR that made the change on Monday.

**Pull request merged**